src/main.rs

```diff
@@ -33,3 +68,15 @@ async fn main() -> anyhow::Result<()> {
     shutdown.await.context("failed to shutdown host")?;
     Ok(())
 }
+
+impl From<Args> for WasmbusHostConfig {
+    fn from(args: Args) -> Self {
+        Self {
+            ctl_nats_url: Url::parse(&format!("nats://{}:{}", args.nats_host, args.nats_port))
+                .expect("failed to parse ctl_nats_url"),
```

Comment:
This should have happened in `main()`, but I don't mind fixing that up in a follow-up myself